src/features/left-sidebar/icons/IconBoxRelay.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import AccessibleSVG from '../../../icons/accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    selected?: boolean,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const iconName = 'icon-box-relay';
+
+const IconBoxRelay = ({ className = '', width = 14, color = '#c4c4c4', selected = false, title }: Props) => (
+    <AccessibleSVG
+        className={classNames(iconName, className, {
```

Comment:
Seems like this should be handled by passing in the appropriate `className` prop from the host component.

Reply:
Again, the purpose of this is to follow the conventions and add a new icon. A refactor seems like a good idea here, however it should be handled as a separate issue ticket and a more thorough review of these icons.